projects/ghostscript/gstoraster_fuzzer_all_colors.cc

```diff
@@ -22,6 +22,6 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
 	data++;
 	size--;
 
-	gs_to_raster_fuzz(data, size, color_scheme);
+	gs_to_raster_fuzz(data, size, color_scheme, "cups");
```

Comment:
If you make a new method, you can let gs_to_raster_fuzz use that method with "cups" added.  Then you don't need to change the existing callers.